ui/partials/main/jobs.html

```diff
@@ -16,8 +16,10 @@
     <span class="result-set-left">
       <span class="result-set-title-left">
         <span>
-          <a href="{{::revisionResultsetFilterUrl}}"
+          <a ng-href="{{::revisionResultsetFilterUrl}}"
+             ng-click="changeResultSet(repoName, resultset.revision)"
              title="View only this resultset"
+             prevent-default-on-left-click
```

Comment:
Can I ask what you mean by "you'd need to either update that scope variable"?  This link click problem turned out to be a nightmare before `prevent-default-on-left-click`; when clicking the link, the following would happen:

(url you're on) -> (revision url with filters) -> (revision url without filters)

What scope variable would make the difference here?